docs/manpage.rst

```diff
@@ -379,6 +379,22 @@ Options controlling ReFrame execution
 
    .. versionadded:: 3.2
 
+.. option:: --distribute=STATE
+
+   Parameterize and run tests on all the nodes of their partitions.
```

Comment:
First, you need to leave a blank line after the first line, so that it serves as a short description of the option. Second, I think you should change this to something more general and elaborate later:

```suggestion
   Distribute the tests on all the nodes in state ``NODESTATE`` in their respective valid partitions.
```